docs/learn/glossary.md

```diff
@@ -99,7 +99,7 @@ Publish/Subscribe (Pub/Sub) is an asynchronous messaging pattern where publisher
 
 A Pub/Sub topic is a string that serves as an identifier for the topic of interest among [GossipSub](#gossipsub) peers. Peers interested in the same topic are likely to maintain a connection and forward messages received on that topic.
 
-### [Rate limit nullifiers](https://rfc.vac.dev/spec/64/#rln-rate-limiting)
+### Rate limit nullifiers
```

Comment:
Is there supposed to be a link here? @LordGhostX

Reply:
The spec was removed from rfc.vac.dev. What if I linked it to https://rate-limiting-nullifier.github.io/rln-docs/ instead?

Reply:
Yeah that would be useful to the reader. I mainly commented because each other heading in the section has a link to an RFC. Your call if you want to add the link.